### PR TITLE
Rollups: Add support for "Always Rollup to Primary Contact"

### PR DESCRIPTION
**Pull request by force2b**
# Critical Changes

# Changes

* Support scenarios where there is no filter group on a rollup definition
* Update the CreateDefaultRollupConfiguration logic to respect the legacy "Enable Soft Credit Rollups" field when creating default rollup metadata

# Issues Closed

# New Metadata

# Deleted Metadata

* Account.Number_of_Opportunities__c

### CODE REVIEW

File: src/classes/CRLP_Batch_Base_Skew.cls [hunk 1]
```diff
@@ -306,6 +316,13 @@ public abstract class CRLP_Batch_Base_Skew extends CRLP_Batch_Base {
                 this.parentIdsNotCommittedWithBatch.put(lastParentIdProcessed, this.batchIteration);
             }
 
+            try {
```
Comment (njjc) [inline src/classes/CRLP_Batch_Base_Skew.cls]: Do we want this in the final product?

Comment (force2b) [reply src/classes/CRLP_Batch_Base_Skew.cls]: No. Though I took that out. Must have accidentally committed instead of removing.

Comment (njjc) [reply src/classes/CRLP_Batch_Base_Skew.cls]: Cool, I'll leave this PR open until you remove it.

**Comment by force2b** [comment]
@njjc FYI - Removed system.debug statements.

**Pull request merged**